doc/build/orm/dataclasses.rst

```diff
@@ -38,7 +38,7 @@ as though it were explicitly decorated using the ``@dataclasses.dataclass``
 decorator.
 
 .. note::  Support for :pep:`681` in typing tools as of **July 3, 2022** is
```

Comment:
could probably update the date too

Reply:
https://github.com/sqlalchemy/sqlalchemy/pull/9463/commits/3f0de536af822b2a78ad5cfb289728b277d92995, added `fully` as it seems other type checkers have basic support (e.g. pyre).